### ISSUE TITLE
Full Realm Exports

### ISSUE THREAD

**Issue by laugirard** [rockcarver/frodo#360]
**Is your feature request related to a problem? Please describe.**
Most times when working with clients, we feel in the need to generate a full realm exports and sometimes a full solution export (with all realms and customizations). While a full realm export is possible with Amster, it is preferred to use just one tool, and Frodo is better.

**Describe the solution you'd like**
I would like to be able to do single and multiple Full-Realm exports.

**Describe alternatives you've considered**
Full-Realm exports must include **everything** from the original realm configuration (services, scripts, agents, clients, journeys, etc.).

**Additional context**
It should be great to have a parameter to select wether to export one realm or all. Also the import command should be able to import a Full-Realm export.

**Comment by laugirard** [comment]
@vscheuber

**Comment by jgdtech** [comment]
I agree. It would be nice to compare differences between environments too (e.g. sandbox to dev, dev to staging, etc).
